holoviews/element/chart.py

```diff
@@ -165,6 +165,10 @@ class Histogram(Element2D):
     vdims = param.List(default=[Dimension('Frequency')], bounds=(1,1))
 
     def __init__(self, values, edges=None, **params):
+        if edges is not None:
+            self.warning("Histogram edges should be supplied as a tuple "
+                         "along with the values, passing the edges will "
+                         "be deprecated in holoviews 2.0.")
```

Comment:
Something to mention in the next changelog/release notes. It will be good to get histogram working consistently with everything else.